src/main/java/com/company/Tweet.java

```diff
@@ -17,15 +20,15 @@ public static void main(String[] args) {
         TwitterFactory tf = new TwitterFactory(cb.build());
         Twitter twitter = tf.getInstance();
         try{
-            if(args[0].length()>280){
+            if(args == null || args.length()==0 || args.length()>280){
                 throw new IndexOutOfBoundsException();
             }
-            Status status = twitter.updateStatus(args[0]);
-            System.out.println(status.getText());
+            Status status = twitter.updateStatus(args);
+            return Response.ok(new TweetResponse(status.getText())).build();
         } catch (IndexOutOfBoundsException e){
-            System.out.println("tweet can only be 1 - 255 characters!");
+            return Response.status(Response.Status.NOT_ACCEPTABLE).entity(new TweetResponse("Tweet must be 1-280 characters!")).build();
```

Comment:
do you actually need to wrap the error in the TweetResponse object?

Reply:
You're throwing an IndexOutOfBoundsException above just to catch it here. Why not directly return the Response at that time?

Reply:
addressed.  returning a Response here makes more sense.  I do not have to wrap the TweetResponse object and can just return a Response status of NOT_ACCEPTABLE.